src/odb/include/odb/db.h

```diff
@@ -3842,6 +3842,18 @@ class dbWire : public dbObject
   ///
   static void destroy(dbWire* wire);
 
+  ///
+  /// get raw data of _dbWire
+  ///
+  void getRawWireData(std::vector<int>& data,
+                      std::vector<unsigned char>& op_codes);
```

Comment:
I assume this APIs can be removed once the antenna checker work finishes and order_wires is no longer required during repair_anntenas.